cmd/manager/main.go

```diff
@@ -196,8 +196,8 @@ func getManagerOptions(scheme *k8sruntime.Scheme, opts options) (manager.Options
 		HealthProbeBindAddress: opts.ProbeAddr,
 		MetricsBindAddress:     "0", // disable controller-runtime managed prometheus endpoint
 		// disable caching of everything
-		ClientBuilder: &newUncachedClientBuilder{},
-		Scheme:        scheme,
+		NewClient: newClient,
```

Comment:
interesting change 👍 

it took a minute for me the get my mind around this change 😄 , `ClientBuilder` is deprecated.
cc @thetechnick (fyi)